src/components/reusable/Graph.js

```diff
@@ -61,48 +67,41 @@ const buildGraph = (data, categories) => {
           style: {
             colors: "#1F302B",
           },
+          formatter: function (value) {
+            return value.toFixed(2); // Set the label value to have a maximum of 2 decimal points
+          },
         },
       },
     },
   };
 };
 
-const Graph = ({ getData, chartData, categories }) => {
-  // Create the graph with the data and categories along with the callback to get more data
-  let chart = buildGraph(chartData, categories);
-  const [selectedFilter, setSelectedFilter] = useState("Today");
-
-  const filters = {
-    DAY: "Today",
-    FIVE: "Past 5 days",
-    MONTH: "Past month",
-    YEAR: "Past Year",
-  };
+const Graph = ({ stockData, getData, xValues, yValues, selectedFilter }) => {
+  // conditional variable to indicate whether stock searched is trending up or down
+  const isTrendingUp = stockData[0].priceChange >= 0;
 
-  const handleFilterSelection = (filter) => {
```

Comment:
so i removed this wrapper function because my thought process was that i could just move the setSelectedFilter to getData, but if its better to have a wrapper function  for this type of stuff for naming lmk